src/vitabel/vitals.py

```diff
@@ -1156,30 +1158,41 @@ def shocks(self):
 
     def compute_etco2_and_ventilations(
         self,
-        mode="threshold",
-        breaththresh: float = 4,
-        etco2_thresh: float = 4,
+        mode: Literal['threshold', 'filter'] = 'filter',
+        breath_thresh: float = 2,
+        etco2_thresh: float = 3,
```

Comment:
If we rename keyword arguments, can we rename them properly? `breath_threshold` and `etco2_threshold`? I don't really see a reason to abbreviate here, and it would improve readability.

(I'd be happy to adapt the remaining occurrences, I won't mark them with comments now.)

```suggestion
        breath_threshold: float = 2,
        etco2_threshold: float = 3,
```